sky/cli.py

```diff
@@ -3179,8 +3180,16 @@ def _output():
                     yield from k8s_realtime_table.get_string()
                     k8s_node_table = _get_kubernetes_node_info_table(context)
                     yield '\n\n'
+                    # TODO(Doyoung): Update the message with the multi-host TPU
+                    # support.
+                    k8s_per_node_acc_message = (
+                        'Kubernetes per node accelerator availability ')
+                    if kubernetes_utils.multi_host_tpu_exists_in_cluster(
+                            context):
+                        k8s_per_node_acc_message += (
+                            '(Note: Multi-host TPUs are not supported.)')
```

Comment:
I'm not quite convinced on why this is needed on top of the additional noting message I added for the following reasons:
1. There can be multiple number of multi-host TPUs. If there's like 10 of them, it should list out all of them. Not sure if this is the best UX as we are trying to keep things concise. If it's important info, we should add them, but..
2. wondering if this is necessary to begin with since the users of TPUs on GKE would know what a multi-host TPU is and its existance in the cluster.

Reply:
My main concern here is that the message `Multi-host TPUs are not supported.` does not convey the meaning of "we excluded some nodes from your cluster", which might confuse the users.

One way is to count the number of nodes / or number of TPUs and show `xxx nodes with multi-host TPU setup is not included / excluded in the resrouces`.

Reply:
@cblmemo I see. That makes sense and I agree to the concern. I extended a message so that the users are noted that the Multi-host TPU nodes in their GKE cluster are excluded from the display.

```
Kubernetes per node accelerator availability (Note: Multi-host TPUs are detected and excluded from the display as multi-host TPUs are not supported.)
NODE_NAME                                  GPU_NAME              TOTAL_GPUS  FREE_GPUS
gke-mix-tpu-dy-default-pool-ad5bdc4d-9lw4  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-bs86  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-cfxn  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-nr5x  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-qgjt  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-rl37  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-v4ts  None                  0           0
gke-mix-tpu-dy-default-pool-ad5bdc4d-zp2x  None                  0           0
gke-tpu-a3716138-984x                      tpu-v5-lite-podslice  4           0
gke-tpu-c5117ac4-qfzt                      tpu-v5-lite-podslice  1           1
```

Reply:
Oh are those TPU in the default pool are TPUs with multi-host? Those looks a little bit overwhelming to me. Is it possible to only show those excluded nodes when `-a`? (It would also be helpful if we could add some comments at the docstr of `sky show-gpus` and here:

https://github.com/skypilot-org/skypilot/blob/42c79e1d0a5e018e275705ada53957573f9a0181/sky/cli.py#L3274-L3275

Reply:
@cblmemo The ones in default pool are not multi host. Multi-host TPUs are not appearing from the list above.

Reply:
I see. Maybe worth filing an issue to emit them in the output as this does not looks relevant to `sky show-gpus`.